frontend/public/style/_overrides.scss

```diff
@@ -213,53 +213,114 @@ tags-input .autocomplete .suggestion-item em {
+  --pf-l-page__sidebar--PaddingTop: 0;
+  .pf-c-nav {
+    // List link
+    --pf-c-nav__list-link--active--after--BackgroundColor: var(--pf-c-nav__list-link--hover--after--BackgroundColor);
+    --pf-c-nav__list-link--active--BackgroundColor: var(--pf-c-nav__list-link--hover--BackgroundColor);
+    --pf-c-nav__list-link--active--Color: var(--pf-c-nav__list-link--hover--Color);
+    --pf-c-nav__list-link--after--Height: 100%;
+    --pf-c-nav__list-link--after--Left: 0;
+    --pf-c-nav__list-link--after--md--Left: var(--pf-c-nav__list-link--after--Left);
+    --pf-c-nav__list-link--after--Width: 3px;
+    --pf-c-nav__list-link--Color: #{$color-pf-black-300};
+    --pf-c-nav__list-link--focus--after--BackgroundColor: var(--pf-c-nav__list-link--hover--after--BackgroundColor);
+    --pf-c-nav__list-link--focus--BackgroundColor: var(--pf-c-nav__list-link--hover--BackgroundColor);
+    --pf-c-nav__list-link--focus--Color: var(--pf-c-nav__list-link--hover--Color);
+    --pf-c-nav__list-link--hover--after--BackgroundColor: transparent;
+    --pf-c-nav__list-link--hover--BackgroundColor: #{$color-pf-black-700};
+    --pf-c-nav__list-link--hover--Color: #{$color-pf-white};
+    --pf-c-nav__list-link--m-current--after--BackgroundColor: #{$color-pf-blue-300};
+    --pf-c-nav__list-link--m-current--Color: var(--pf-c-nav__list-link--hover--Color);
+    --pf-c-nav__list-link--md--PaddingLeft: 32px;
+    --pf-c-nav__list-link--PaddingBottom: var(--pf-c-nav__list-link--PaddingTop);
+    --pf-c-nav__list-link--PaddingLeft: 16px;
+    --pf-c-nav__list-link--PaddingTop: 17px;
 
-.pf-c-nav__link {
-  --pf-c-nav__list-link--Color: #{$color-pf-black-300};
-}
+    // Simple List
+    --pf-c-nav__simple-list-link--active--BackgroundColor: var(--pf-c-nav__list-link--hover--BackgroundColor);
+    --pf-c-nav__simple-list-link--active--Color: var(--pf-c-nav__list-link--hover--Color);
+    --pf-c-nav__simple-list-link--Color: var(--pf-c-nav__list-link--Color);
+    --pf-c-nav__simple-list-link--focus--BackgroundColor: var(--pf-c-nav__list-link--hover--BackgroundColor);
+    --pf-c-nav__simple-list-link--focus--Color: var(--pf-c-nav__list-link--hover--Color);
+    --pf-c-nav__simple-list-link--focus--FontWeight: var(--pf-global--FontWeight--normal);
+    --pf-c-nav__simple-list-link--m-current--BackgroundColor: var(--pf-c-nav__list-link--hover--BackgroundColor);
+    --pf-c-nav__simple-list-link--m-current--Color: var(--pf-c-nav__simple-list-link--active--Color);
+    --pf-c-nav__simple-list-link--PaddingBottom: 2px;
+    --pf-c-nav__simple-list-link--PaddingRight: 5px;
+    --pf-c-nav__simple-list-link--PaddingTop: 4px;
 
-.pf-c-nav__list {
-  .pf-c-nav__link {
-    &:active {
-      --pf-c-nav__list-link--active--Color: #{$color-pf-white};
+    // Subnav
+    --pf-c-nav__subnav--MarginTop: 0;
+    --pf-c-nav__subnav--MaxHeight: 100%;
+    --pf-c-nav__subnav-item--PaddingLeft: var(--pf-c-nav__list-link--PaddingLeft);
+    @media screen and (min-width: 768px) {
```

Comment:
Recommend `$grid-float-breakpoint` instead of `768px`

Reply:
updated